### ISSUE TITLE
The detection results of  yolov8 and Deformable DETR

### ISSUE THREAD

**Issue by Kroery** [dyhBUPT/iKUN#32]
Hi, thanks for your excellent work!

Could you provide the detection results from YOLOv8 and Deformable DETR?

**Comment by dyhBUPT** [comment]
Hi, sorry but the detection results are not prepared well. Maybe it's not too hard to train detectors following the official repo.

**Comment by Kroery** [comment]
Hi, I understand that the detection results are not available. Would it be possible for you to share the config files for training YOLOv8 and Deformable DETR? That would be really helpful for me to replicate the setup.

**Comment by dyhBUPT** [comment]
oh, of course.

For YOLOv8, we use the COCO-pretrained YOLOv8-L, and finetune it on Refer-KITTI train set for only 1 epoch. The input size is 1300x400, the batch size is 8, the lr scheduler is cosine. Four classes are trained, i.e., car, pedestrian, other vehicles, other persons.

For DeformableDETR, we directly modify the source code of TransRMOT to make it only perform detection training and inference. It may be the fastest way to realize a strong detector on Refer-KITTI.

Because the codes of both YOLOv8 and TransRMOT are organized well, I think it's not very hard for you to train good detectors. Good luck!

**Comment by xzb666** [comment]
@Kroery Sorry to bother you, but have you reproduced the results related to Deformable DETR in the NeuralSORT
